### PR TITLE
first proposal to improve the README

### PR DESCRIPTION
**Pull request by GohuHQ**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -39,4 +39,107 @@ applications on top of SQLAlchemy.
...
+
+Installation will add the ``anyblok`` commands to the environment.
+
+.. note:: AnyBlok use Python version >= 3.5
+
+Running Tests
+-------------
+
+.. .. seealso:: the :ref:`section about testing of AnyBlok applications
+..              <basedoc_tests>`.
```
[the remark this answers is not in the document]
Comment (GohuHQ) [reply README.rst]: Yes most of it is extracted from http://doc.anyblok.org/en/latest/FRONT.html.
I'll fix it.

**Comment by GohuHQ** [comment]
Thanks @petrus-v for the feedback, i pushed a new version.

File: README.rst [hunk 2]
```diff
@@ -39,4 +39,99 @@ applications on top of SQLAlchemy.
...
-See the `latest documentation <http://doc.anyblok.org/>`_
+AnyBlok is hosted on `github <https://github.com>`_ - the main project
+page is at https://github.com/anyblok/anyblok or
+http://code.anyblok.org. source code is tracked here
+using `git <https://git-scm.com>`_.
+
+Releases and project status are available on Pypi at
+https://pypi.python.org/pypi/anyblok.
+
+<<<<<<< HEAD
```
Comment (petrus-v) [inline README.rst]: oubli de résolution de conflit ?

File: README.rst [hunk 3]
```diff
@@ -39,4 +39,107 @@ applications on top of SQLAlchemy.
...
+    anyblok_createdb --db-name test_anyblok --db-driver-name postgresql --install-all-bloks
+    ANYBLOK_DATABASE_DRIVER=postgresql ANYBLOK_DATABASE_NAME=test_anyblok py.test anyblok/bloks
+
+AnyBlok is tested continuously using `Travis CI
+<https://travis-ci.org/AnyBlok/AnyBlok>`_
+
+Dependencies
+------------
+
+AnyBlok works with **Python 3.5** and later. The install process will
```
[the remark this answers is not in the document]
Comment (petrus-v) [reply README.rst]: I don't know too... at least removing 3.5 in setup.py https://github.com/AnyBlok/AnyBlok/blob/master/setup.py#L86

File: README.rst (hunk 2 above)
Comment (GohuHQ) [reply README.rst]: Chelou, ça a dû faire ça au moment du rebase :/
Merci !

File: README.rst [hunk 4]
```diff
@@ -39,4 +39,90 @@ applications on top of SQLAlchemy.
...
+The most recent published version of the documentation should be at
+https://doc.anyblok.org.
+
+There is a tutorial to teach you how to develop applications with AnyBlok at
+https://anyblok.gitbooks.io/anyblok-book/content/en/
+
+Project Status
+--------------
+
+AnyBlok is currently in alpha status and is expected to be fairly
```
Comment (petrus-v) [inline README.rst]: @jssuzanne do you consider we are still in alpha status ?

File: README.rst [hunk 5]
```diff
@@ -39,4 +39,90 @@ applications on top of SQLAlchemy.
...
+    anyblok_createdb --db-name test_anyblok --db-driver-name postgresql --install-all-bloks
+    ANYBLOK_DATABASE_DRIVER=postgresql ANYBLOK_DATABASE_NAME=test_anyblok py.test anyblok/bloks
+
+AnyBlok is tested continuously using `Travis CI
+<https://travis-ci.org/AnyBlok/AnyBlok>`_
+
+Contributing (hackers needed!)
+------------------------------
+
+AnyBlok is at a very early stage, feel free to fork, talk with core dev, and spread the word!
```
Comment (petrus-v) [inline README.rst]: @jssuzanne how early stage is it ? we may afraid developer that this project is not ready for production usage

**Comment by petrus-v** [comment]
Thanks @GohuHQ LGTM as it, open discussion could be in an other PR if you prefer, I was just wondering the state of the project, I agree to say it's quite young framework but would like to mention it's ready for production usage as it does !

@jssuzanne [roads to release](https://github.com/AnyBlok/AnyBlok/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0) v1.0 looks close, isn't it ? should we plan a sprint focus on that ?

**Comment by jssuzanne** [comment]
You should also fix setup.py, because the front and the readme are sent on pypi, or you duplicate front on readme

**Comment by GohuHQ** [comment]
Ok @jssuzanne . I'll duplicate the content because we want this content on AnyBlok documentation i guess.

**Comment by jssuzanne** [comment]
Yes we may. just keep in mind this duplication

File: doc/FRONT.rst [hunk 6]
```diff
@@ -105,20 +108,17 @@ Jean-Sébastien Suzanne
 Contributors
 ------------
 
-`Anybox <http://anybox.fr>`_ team:
-
+* Jean-Sébastien Suzanne
```
[the remark this answers is not in the document]
Comment (GohuHQ) [reply doc/FRONT.rst]: It's already done. Please check the last version at https://github.com/GohuHQ/AnyBlok/tree/adding_informations_to_the_readme

**Pull request merged**